### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by q320d** [squint-cljs/cherry#125] [General]
This project is awesome cljs needs a modern runtime supporting async/await! What is missing for it to be usable in production? Is the intention to bring it to a production-ready state or will it remain an experiment? Thanks!

> ⚠️ This project is an experiment and not recommended to be used in production. It currently has many bugs and will undergo many breaking changes.

**Comment by borkdude** [comment]
This project is already being used in production, I think I can remove this warning soon
